src/styles/card/card.scss

```diff
@@ -0,0 +1,92 @@
+
+.card {
+  height: 406px;
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $background-color;
+  display: flex;
+  flex-wrap: wrap;
+
+  &__photo {
+    height: 134px;
+    width: 160px;
+    margin: 33px 20px 0;
+  }
+
+  &__description {
+    margin: 35px 16px 0;
+    color: $primary-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin: 4px 0 0 16px;
+    color: $secondary-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    width: $card-width;
+    height: 18px;
+    display: flex;
+    justify-content: space-between;
+    margin-top: 7px;
+
+    &--text {
+      color: $secondary-color;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      margin-left: 17px;
+    }
+
+    &--value {
+      margin-left: 50px;
+      color: $primary-color;
+      text-align: right;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    width: $card-width;
+    height: 40px;
+    margin: 16px;
+    border-radius: 5px;
+    background: $blue-link-color;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__button:hover {
+    border-radius: 5px;
+    border: 1px solid $blue-link-color;
+    background: $background-color;
+  }
+
+  &__link {
+    color: white;
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    text-decoration: none;
+  }
+
+  &__link:hover {
+    color: $blue-link-color;
```

Comment:
The properties color, text-align, font-size, font-weight, and text-decoration are duplicate of the same properties in &__link. It is not necessary to repeat these properties in &__link:hover, you can remove these duplicate properties. Only keep the properties that change on hover.